styles/containers.css

```diff
@@ -0,0 +1,387 @@
+:root {
+    --bs-blue: #91C8FF;
+    --bs-indigo: #6610f2;
+    --bs-purple: #6f42c1;
+    --gray-body: #F7FAFC;
+    --aside: #F9FAFB;
+}
+
+h2 {
+    margin: 1rem;
+}
+
+hr {
+    margin-top: 2rem;
+    margin-bottom: 1rem;
+    border: 1px solid black;
+    opacity: 0;
+}
+
+.list-item {
+    list-style: none;
+    padding-top: 1rem;
+    padding-left: 1.5rem;
+}
+
+.list-item-link {
+    text-decoration: none;
+    color: black;
+    font-size: 1.5rem;
+    cursor: pointer;
+    font-weight: 600;
+}
+
+.list-item-link:hover {
+    background: var(--bs-blue);
+    border-radius: 10px;
+    z-index: 1;
+    padding: 10px;
+}
+
+/* Header Container */
+
+.header-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+}
+
+.list .list-items {
+    list-style: none;
+    display: inline;
+    margin: 0px 20px 0px 0px;
+    font-size: large;
+}
+
+.link-item {
+    text-decoration: none;
+    color: var(--text);
+}
+
+.active-link {
+    background: #60A5FA;
+    padding: 0.5rem;
+    border-radius: 3px;
+}
+
+#header-title {
+    margin: 3rem 0rem 3rem 1rem;
```

Comment:
Your header logo and nav links are a little below out of the header 
You can fix that by making the below changes in this line
 margin: **1rem** 0rem 3rem 1rem;

Reply:
Thanks, will replicate the changes.